kitsune/wiki/templates/wiki/translate.html

```diff
@@ -171,7 +171,7 @@
             <div class="approved">
               <h3>{{ _('Approved {default_locale} version:')|f(default_locale=settings.LANGUAGES_DICT[settings.WIKI_DEFAULT_LANGUAGE.lower()]) }}</h3>
               <div id="content-or-diff" class="content">
-                <textarea readonly="readonly">{{ based_on.content }}</textarea>
+                <textarea readonly="readonly" dir="ltr">{{ based_on.content }}</textarea>
```

Comment:
Might be worth adding a `WIKI_DEFAULT_DIRECTION` to match `WIKI_DEFAULT_LANGUAGE`.